Booting/linux-bootstrap-1.md

```diff
@@ -97,7 +101,7 @@ SECTIONS {
 }
 ```
 
-Now BIOS has started to work. After all initializations and hardware checking, it needs to load operating system. BIOS tries to find bootable device which contains boot sector. Boot sector is the first sector on device (512 bytes) and contains sequence of `0x55` and `0xaa` at 511 and 512 byte. For example:
+Now BIOS has started to work. After all initializations and hardware checking, it needs to load the operating system. BIOS tries to find a bootable device which contains a boot sector. A boot sector is the first sector on the device (512 bytes) and contains a sequence of `0x55` and `0xaa` at 511 and 512 byte. For example:
```

Comment:
I think this paragrahp is overloaded with articles, I'd suggest to change it like this:

Now BIOS has started to work. After all initializations and hardware checking, it needs to load a operating system. BIOS tries to find a bootable device which contains boot sector. Boot sector is the first sector on device (512 bytes) and contains the sequence of `0x55` and `0xaa` at 511 and 512 bytes. For example: